addons/l10n_br_website_sale/models/website.py

```diff
@@ -0,0 +1,13 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        for website in vals_list:
+            if self.env['res.company'].browse(website['company_id']).country_code == "BR":
+                website['show_line_subtotals_tax_selection'] = 'tax_included'
+        return super().create(vals_list)
```

Comment:
Please handle that properly in the `res.config.settings` model and not here.
Furthermore, this breaks a lot of tests in nightly because `company_id` is not always specified and your code should not break even when it isn't ...

https://runbot.odoo.com/runbot/build/58840232

Reply:
As discussed in PM, this will remain in the website model but the company_id part will ofc get fixed.

While discussing, a consistency issue has been highlighted. This will be also fixed.

Those fix are in the following PR : https://github.com/odoo/odoo/pull/156096